src/scss/variables.scss

```diff
@@ -4,5 +4,5 @@
 $border-radius: 0.25rem;
 $default-padding: 10px;
 $light-gray: $gray-200;
-$text-green: green;
-$text-red: red;
+$text-green: $green;
```

Comment:
Could we just use `$green` instead of `$text-green` everywhere now?

Reply:
I suppose we could yes